### PR TITLE
Create column actions menu with Close Column option

### PR DESCRIPTION
**Pull request by davidchristie**

### CODE REVIEW

File: src/containers/Column/Actions/Actions.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <Dropdown
+          button
+          floating
+          icon={null}
+          labeled
+          options={this.props.actions}
+          trigger={trigger}
+        />
+      </Button.Group>
+    )
```
Comment (amandahogan) [inline src/containers/Column/Actions/Actions.js:22]: Does this need to be a dropdown?, I guess it's better so people don't accidentally delete columns, or cards, but it seems to be redundant for one action at the moment.

File: src/containers/Column/Column.js [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
   render () {
+    const actions = [
+      {
+        onClick: this.props.closeThisColumn,
+        text: 'Close Column'
+      }
+    ]
     return (
       <Container>
+        <Actions actions={actions} />
```
Comment (amandahogan) [inline src/containers/Column/Column.js:23]: Lots of actions in one line... but I can't really think of a better alternative word to use, I know saw Trello uses List actions, and I thought of maybe tasks but doesn't really effect the code.

**Review by amandahogan — APPROVED** [review]

File: src/containers/Column/Actions/Actions.js (hunk 1 above)
Comment (davidchristie) [reply src/containers/Column/Actions/Actions.js:22]: @amandahogan Could definitely just be a button. I went with a dropdown to mimic Trello. Plus I was imagining a bunch of other useful Trello actions we could eventually list in the same menu - Follow All, Unfollow All, Archive This List etc.

File: src/containers/Column/Column.js (hunk 2 above)
Comment (davidchristie) [reply src/containers/Column/Column.js:23]: @amandahogan  `<Options options={options} />` ?

**Review by Quinn-H — APPROVED** [review]

File: src/containers/Column/Column.js (hunk 2 above)
Comment (amandahogan) [reply src/containers/Column/Column.js:23]: yeah I think I like options better then actions :) easily changed.

**Pull request merged**